machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -1537,38 +1290,6 @@ export function selectId(state: State) {
   return state.context.vcMetadata.id;
 }
 
-export function selectIdType(state: State) {
-  return state.context.vcMetadata.idType;
-}
-
-export function selectCredential(state: State) {
-  return state.context.credential;
-}
-
-export function selectIsOtpError(state: State) {
-  return state.context.otpError;
-}
-
-export function selectIsLockingVc(state: State) {
-  return state.matches('lockingVc');
-}
-
-export function selectIsRevokingVc(state: State) {
-  return state.matches('revokingVc');
-}
-
-export function selectIsLoggingRevoke(state: State) {
-  return state.matches('loggingRevoke');
-}
-
-export function selectIsAcceptingOtpInput(state: State) {
-  return state.matches('acceptingOtpInput');
-}
```

Comment:
Is this not used anywhere in the first time when the VC is being downloaded?

Reply:
@vharsh selectIsAcceptingOtpInput selector for download is taken from AddVCModalMachine.ts